tests/Dialect/Secret/Conversions/secret_to_ckks/type_converter.mlir

```diff
@@ -1,7 +1,7 @@
 // Tests that the type converter handles both converting tensors to SIMD slots
 // if aligned, or to tensors of ciphertext.
 
-// RUN: heir-opt --mlir-print-local-scope --secret-insert-mgmt-ckks=include-first-mul=false --secret-distribute-generic --canonicalize --secret-to-ckks --split-input-file %s | FileCheck %s
+// RUN: heir-opt --mlir-print-local-scope --secret-insert-mgmt-ckks=include-first-mul=false --annotate-mgmt --secret-distribute-generic --canonicalize --secret-to-ckks --split-input-file %s | FileCheck %s
```

Comment:
This is OK, but if you want the annotations to be an invariant of the passes that require them, you could run them as a sub-pipeline (similar to what you did running CSE at the end of SecretInsertMgmtBGV) either before a pass that requires them, or at the end of a pass that invalidates them. Then there is no risk of someone forgetting to insert the annotate-mgmt pass.